source/index.md

```diff
@@ -150,6 +150,7 @@ Or just write Fortran software for your research, business, or schoolwork. You c
 :::{toctree}
 :hidden:
 
+Playground <https://play.fortran-lang.org/>
```

Comment:
```suggestion
Play <https://play.fortran-lang.org/>
```

Rationale:

* The navbar is already busy and "Playground" is a mouthful
* "Play" is a verb so it has a more active voice;
* It flows better next to "Learn"

Reply:
I was looking at the Rust page where they use *Playground*, but I like *Play* better. I'm not sure whether we can insert an actual triangle here (maybe with UTF8?) to make it stand out more.